lisa/tools/meson.py

```diff
@@ -30,9 +30,25 @@ def can_install(self) -> bool:
     def _install(self) -> bool:
         posix_os: Posix = cast(Posix, self.node.os)
         # use pip to make sure we install a recent version
-        if (not posix_os.package_exists("meson")) or posix_os.get_package_information(
-            "meson", use_cached=False
-        ) < "0.52.0":
+
+        package_installed = ""
+        package_available = ""
+        for pkg in ["meson", "python3-meson"]:
+            if (
+                posix_os.package_exists(pkg)
+                and posix_os.get_package_information(pkg, use_cached=False) >= "0.52.0"
+            ):
+                package_installed = pkg
```

Comment:
The logic is unclear to me. Does it mean either mason or python3-meson installed, it means installed? Please add code comments to explain the logic, if it's true.